### PR TITLE
Gp 2221

### PR DESCRIPTION
**Pull request by xuemeiluo**

### CODE REVIEW

**Comment by xuemeiluo** [comment]
retest this please

File: bwaMem.wdl [hunk 1]
```diff
@@ -0,0 +1,487 @@
...
+            url: "https://github.com/OpenGene/slicer/archive/v0.3.0.tar.gz"
+        }
+      ]
+    }
+
+    output {
+        File bwaMemBam = bamMerge.outputMergedBam
+        File bwaMemIndex = indexBam.outputBai
+        File? log = adapterTrimmingLog.summaryLog
+        File? AllLogs = adapterTrimmingLog.allLogs
```
Comment (apmasell) [inline bwaMem.wdl]: Please make this camel case.

File: bwaMem.wdl [hunk 2]
```diff
@@ -0,0 +1,487 @@
...
+        timeout:   "Hours before task timeout"
+    }
+
+    String allLog = "~{outputFileNamePrefix}.txt"
+    String log = "~{outputFileNamePrefix}.log"
+
+    command <<<
+        set -e
+        set -o pipefail
+        awk 'BEGINFILE {print "###################################\n"}{print}' ~{sep=" " inputLogs} > ~{allLog}
```
Comment (apmasell) [inline bwaMem.wdl]: Umm... this should probably be it's own program. Having something to summarise cutadapt logs would be useful on its own and ideally do it in a way without so much AWK.

**Comment by xuemeiluo** [comment]
retest this please

**Comment by xuemeiluo** [comment]
retest this please

**Review by pruzanov — APPROVED** [review]

**Review by pruzanov — APPROVED** [review]

**Comment by xuemeiluo** [comment]
retest this please

**Review by mlaszloffy — APPROVED** [review]

**Review by apmasell — APPROVED** [review]
Given awk stuff will be addressed in the next future.

File: bwaMem.wdl (hunk 1 above)
Comment (xuemeiluo) [reply bwaMem.wdl]: have modified it

**Pull request merged**